src/js/index.ts

```diff
@@ -31,6 +33,15 @@ store.subscribe(listeners.createStoreListener(store, ctrl));
 );
 
 (store as any).dispatch(addMark(Mark('group', {_parent: 1})));
+var state = store.getState();
+var group = state.getIn(['vis', 'present', 'marks', '2']);
+(store as any).dispatch(addLayout(Layout({
+  rows: 1,
+  cols: 1,
+  groups: [2],
+  rowSizes: [group.encode.update.height],
+  colSizes: [group.encode.update.width]
+}))); // fix hardcoded group ID
```

Comment:
i think the `addGroup` action you added solves this problem:

```
(store as any).dispatch(addLayout(Layout({
  rows: 1,
  cols: 1,
  groups: [],
  rowSizes: [],
  colSizes: []
})));
(store as any).dispatch(addGroup(Mark('group', {_parent: 1})));
```

addGroup will handle adding the group to the layout